cobalt/black_box_tests/testdata/pointer_event_test_utils.js

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file provides test utility functions for pointer event related
+// tests.
+
+// Fail if the test is not finished within 30 seconds.
+const kTimeout = 30 * 1000;
+setTimeout(fail, kTimeout);
+
+function fail() {
+    console.log("Failing due to timeout!");
+    assertTrue(false);
+}
+
+function phaseName(phase) {
+    switch (phase) {
+        case 0: return 'none';
+        case 1: return 'capturing';
+        case 2: return 'at target';
+        case 3: return 'bubbling';
+    }
+    return `[unknown: '${phase}']`;
+}
+
+let current_event_number = 0
+let failure_count = 0;
+
+function checkEvent(e) {
```

Comment:
Did https://github.com/youtube/cobalt/pull/1661#discussion_r1344770860 not work for this? I tried to update it to work, and even if there's something wrong I don't think it should be hard to fix?

Reply:
Relied in the original comment: https://github.com/youtube/cobalt/pull/1661#discussion_r1346369486. I think it actually makes the code more complicated dealing with the last_event.